### DISCUSSION TITLE
📝Comments of [programmers/120866]

### DISCUSSION THREAD

**Discussion by giscus** [PresentKim/legacy-blog-sveltekit#12] [Announcements]
# 📝Comments of [programmers/120866]

지뢰가 있는 지역과 지뢰에 인접한 위, 아래, 좌, 우 대각선 칸을 모두 위험지역으로 분류합니다. 지뢰는 2차원 배열 board에 1로 표시되어 있고 board에는 지뢰가 매설 된 지역 1과, 지뢰가 없는 지역 0만 존재합니다. 지뢰가 매설된 지역의 지도 board가 매개변수로 주어질 때, 안전한 지역의 칸 수를 return하도록 solution 함수를 완성해주세요.

https://present.kim/posts/programmers/120866

**Comment by PresentKim** [comment]
## 숏코딩 (Code Golf)
```javascript
solution=_=>_.flatMap((r,y)=>r.map((v,x)=>{for(i=-1;i<2;++i)for(j=-1;j<2;++j)if(_[y+i]?.[x+j])return 0;return 1})).filter(v=>v).length
```
